scripts/ldmx-env.sh

```diff
@@ -66,5 +66,5 @@ docker pull ${_docker_hub_repo}:${LDMX_DOCKER_TAG}
 #   the container runs the users command from the same place that the user
 #   intended.
 ###############################################################################
-alias ldmx='docker run -i -e LDMX_BASE -v $LDMX_BASE:$LDMX_BASE -u $(id -u ${USER}):$(id -g ${USER}) ${_docker_hub_repo}:$LDMX_DOCKER_TAG $(pwd)'
+alias ldmx='docker run -it -e LDMX_BASE -v $LDMX_BASE:$LDMX_BASE -u $(id -u ${USER}):$(id -g ${USER}) $LDMX_DOCKER_TAG $(pwd)'
```

Comment:
Forgot the `t` before. This is not connected to this PR, but it is helpful when using the docker container because then the user can see the colors and use ctrl+C.